.github/workflows/docker-build-test.yml

```diff
@@ -17,20 +17,22 @@ jobs:
 
       - name: Docker Metadata
         id: docker_metadata
-        uses: docker/metadata-action@v5.3.0
+        uses: docker/metadata-action@v5
         with:
           images: catalystcoop/pudl-etl
           flavor: |
             latest=auto
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3.0.0
+        uses: docker/setup-buildx-action@v3
 
       - name: Build image but do not push to Docker Hub
-        uses: docker/build-push-action@v5.1.0
+        uses: docker/build-push-action@v5
         with:
           context: .
           file: docker/Dockerfile
           push: false
           cache-from: type=gha
           cache-to: type=gha,mode=max
+          secrets: |
+            "GITHUB_REF=${{ github.ref_name }}"
```

Comment:
Is there no way to just pass an environment variable in?

https://docs.docker.com/build/ci/github-actions/secrets/